modules/API/API.js

```diff
@@ -541,6 +606,22 @@ function initCommands() {
         case 'is-audio-muted':
             callback(APP.conference.isLocalAudioMuted());
             break;
+        case 'is-moderation-on': {
+            const { mediaType } = request;
+            const type = mediaType === MEDIA_TYPE.VIDEO ? mediaType : MEDIA_TYPE.AUDIO;
```

Comment:
`const type = mediaType  || MEDIA_TYPE.AUDIO;` . Or do we expect other than an actual media type or undefined? The current implementation would hide any typo in the media type.